e2e/test-component/scenarios/embedding-sdk/interactive-question.cy.spec.tsx

```diff
@@ -457,9 +457,17 @@ describe("scenarios > embedding-sdk > interactive-question", () => {
 
     getSdkRoot().within(() => {
       cy.findByText(`id = ${FIRST_COLLECTION_ENTITY_ID}`).should("exist");
+
+      cy.log("click on the button to switch target collection");
       cy.findByText("use second collection").click();
       cy.findByText(`id = ${SECOND_COLLECTION_ENTITY_ID}`).should("exist");
 
+      cy.log("close any existing open popovers to reduce flakes");
+      cy.findByText("Data").click();
```

Comment:
Without this, 1 in 20 run will flake due to the popover still not being closed yet from the previous render.

Reply:
@npretto it does work! Tried with 20 runs and I don't see any flake. Lemme commit that 🙏🏻 

![CleanShot 2568-05-15 at 16 59 26@2x](https://github.com/user-attachments/assets/7f8aed92-d985-4a73-8ef2-d9a0a543c72d)

Reply:
Fixed in https://github.com/metabase/metabase/pull/58014/commits/eb3cfda4911966fbc3cad41c48594783478b0f38, thanks!